File: PYTHON/Daily Flash/week1/Day2_Jan14/code_python/program1.py
'''
Program 1: Write a Program to check whether the Character is Alphabet or not.
Input: v
Output: v is an alphabet.
'''

char = input("Enter one character: ")

if( (char >= 'a' and char <= 'z') or (char >='A' and char <='Z' ) ):
        print(char ," is an Alphabet")
else:
        print(char ," is not an Alphabet")

File: JAVA/Daily Flash Codes/Week1/19August/program1.java
/*
 
Program 1: Write a Program to check whether the Character is Uppercase or
Lowercase Character.
Input: v
Output: letter v is in lowercase
 */

import java.io.*;

class Program {
	public static void main(String args[]) throws IOException {
		char c;
		System.out.println("Enter Character: ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in) );
		c = (char)br.read();		//to read only one character


		if( (c >=97 && c<=122) || (c >=65 && c <=90) ) {
	
			if(c >=97 && c<=122)
				System.out.println("Letter "+ c + " is in Lower case");
			else
				System.out.println("Letter "+ c + " is in upper case");
		}
		else
			System.out.println("Letter "+ c + " is not an alphabet");
	}
}
